### ISSUE TITLE
terrible output after 2k itr

### ISSUE THREAD

**Issue by Continue7777** [NVlabs/stylegan2-ada-pytorch#157]
what happen here,and what can i do?

dataset:deepfashion landmark filter about 1.6w.
cfg:default

itr:2k
![image](https://user-images.githubusercontent.com/20719321/128481198-c6bd02f4-1071-4b68-ae5e-0afdf6c89e86.png)

itr:3k
![image](https://user-images.githubusercontent.com/20719321/128481161-6b3fd0fc-ee54-4c94-a760-ffc713a6c94d.png)

about loss
![image](https://user-images.githubusercontent.com/20719321/128481004-18059950-5945-4612-a7da-ecd77d746b15.png)
![image](https://user-images.githubusercontent.com/20719321/128481114-d2dacb3e-17ff-4371-9412-3b382c12cee8.png)
![image](https://user-images.githubusercontent.com/20719321/128481134-ba376c6b-7b43-45f9-afc1-5aa73b1c560a.png)

**Comment by thusinh1969** [comment]
Same. After 4,000 itr, FiD wildly jumps all over the places, from 58 to 634 and hanging over there ... Shall we stop training and do it again ? What is going on ?

Steve

**Comment by vsemecky** [comment]
Since it looked pretty good in 2 kimg, I'd try to slow it down. What learning rate did you use?

I often use LR 0.003, but when the network becomes unstable, I need to lower it to 0.002 or even 0.001.
